public/cli/lib/commands/ffmpeg.ts

```diff
@@ -9,21 +9,69 @@ import { paths } from '../path-constants';
 
 export async function combineFrames(settings: FfmpegSettings): Promise<void> {
   const executeAudioPath = await combineAudioIfNecessary(settings.audioFiles);
-  //Arguments for ffmpeg
-  const args = [
-    '-framerate',
-    settings.framerateIn.toString(),
-    '-loglevel',
-    'error',
-    '-i',
-    path.join(settings.imagesPath, 'frame_%06d.png'),
-  ];
+  let audioLength = 1;
+  const args = ['-v', 'error'];
+
+  // calculate the audio duration
```

Comment:
Often comments like this indicate that this section of code could be extracted into its own method. As it turns out we already have a method that does this `getAudioDurationInMilliseconds` in `hearThisImport.ts`. Do they do the same job? Can you combine their use?